src/quo/components/notifications/toast/view.cljs

```diff
@@ -69,15 +69,21 @@
 (def ^:private toast-container (quo.theme/with-theme toast-container-internal))
 
 (defn toast
-  [{:keys [icon icon-color title text action undo-duration undo-on-press container-style theme user]}]
-  (let [context-theme (or theme (quo.theme/get-theme))]
+  "Options:
+   
+   :type => :neutral/:negative/:positive
+   "
+  [{:keys [type title text action undo-duration undo-on-press container-style theme user]
+    :or   {type :neutral}}]
+  (let [context-theme (or theme (quo.theme/get-theme))
+        icon-name     (case type
+                        :positive :i/incorrect
+                        :negative :i/correct
+                        :neutral  :i/placeholder)]
```

Comment:
oh it seems unlikely that we are using :i/placeholder as the neutral icon and this is something that can be overwritten? - can you verify with design how this should be done? 🙏

Reply:
I confirmed that it's the placeholder.

Reply:
I just saw that message. They said it depends on context, so for neutral it can change 👍

Reply:
@codemaster115 please review this comment

Reply:
@J-Son89 It's resolved now.